Documentation/docs/appendix.md

```diff
@@ -225,9 +225,55 @@ If the SIMPLE_EXTRUSION is used, bottom topography can be optionally added to th
 		eqn = f(x,y) = some function of (x,y) as an equation
 	\end{TOPOGRAPHY}
 
-
-
-
-
-
-
+## Appendix E: ABAQUS mesh file format<a name="ABAQUS"></a>
+The ABAQUS mesh file which should use the extension `.inp` is divided into two parts. The top portion is the standard [Abaqus file format](https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-model.htm). The skeleton of this form for a quadrilateral mesh with 122 corner nodes and 103 elements is as follows:
+
+	*Heading
+ 	 File created by HOHQMesh
+	*NODE
+	1, x1, y1, z1
+	2, x2, y2, z2
+	.
+	.
+	.
+	122, x122, y122, z122
+	*ELEMENT, type=CPS4, ELSET=Surface1
+	1, 1, 2, 9, 8
+	2, 2, 3, 10, 9
+	.
+	.
+	.
+	103, 7, 122, 87, 8
+
+Note that the first column in the node or element list is used for indexing purposes. The four other indices in each line of the *ELEMENT list
+are the corner IDs that dictate the element connectivity. These corner IDs are listed to guarantee right-handed element just as with the ISM format.
+The Abaqus element type "CPS4" corresponds to a quadrilateral element. For the three-dimensional variant of this mesh file output we use
+
+	*ELEMENT, type=C3D8, ELSET=Volume1
+
+where "C3D8" corresponds to a hexahedron.
+
+The standard Abaqus file format can be used to create a straight-sided mesh. The high-order boundary information and curvature
+generated by HOHQMesh is output in the second portion of this mesh file. We demarcate between the two portions of the mesh file with the comment
+line
+
+	** ***** HOHQMesh boundary information ***** **
+
+In the Abaqus format anything prefaced with `** ` is treated as a comment and is ignored by an ABAQUS file parser. Therefore, all the HOHQMesh
```

Comment:
Is it really `** ` or just `**`? At least according to https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-inputsyntax.htm:
> Any line that begins with stars in columns 1 and 2 (**) is a comment line.

Reply:
Technically I think it is only two `**`. But  If I do not include the extra space after `**` (especially on the real numbers / integers) then the `AbaqusReader.jl` and p4est throw errors.